.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, W503
+ignore = E203, E266, W503, E501
```

Comment:
you still need to delete `E501`

Reply:
It is possible to bypass the 501 error through additional variables. Discussed on QNA. In this particular case, it is better to add an exception inside the flake8